devdocker

```diff
@@ -127,12 +127,25 @@ def dockerImageBuild(args):
   return subprocess.check_output(
       ['docker', 'image', 'build', '-q'] + args).strip()
 
+def extendCfg(cfg, myCfg):
+  def getList(o, name):
+    return getattr(o, name, [])
+  def getDict(o, name):
+    return getattr(o, name, {})
+  cfg.mount = getList(cfg, 'mount') + getList(myCfg, 'mount')
+  cfg.ports = getList(cfg, 'ports') + getList(myCfg, 'ports')
+  cfg.environment = dict(getDict(cfg, 'environment'), \
+                         **getDict(myCfg, 'environment'))
+  cfg.postCreateCmds = getList(cfg, 'postCreateCmds') + \
+    getList(myCfg, 'postCreateCmds')
+  return cfg
```

Comment:
as we discussed, we could keep this on the back-burner for a while until the need becomes clearer.